client/checkout/woopay/express-button/test/woopay-express-checkout-button.test.js

```diff
@@ -122,10 +191,29 @@ describe( 'WoopayExpressCheckoutButton', () => {
 		} );
 		userEvent.click( expressButton );
 
-		expect( expressCheckoutIframe ).not.toHaveBeenCalled();
+		await waitFor( () => {
+			expect( request ).not.toHaveBeenCalled();
+			expect( expressCheckoutIframe ).not.toHaveBeenCalled();
+		} );
 	} );
 
 	describe( 'Product Page', () => {
+		test( 'does not prefetch session data by default', async () => {
```

Comment:
Should we add a test to ensure the session data is fetched when the WooPay button is clicked?

Reply:
I've tried that but I'm not sure if that's feasible. The thing is that the logic to fetch the session data is set _after_ the iframe loads and I don't know if there's a way for us to simulate that iframe being loaded through Jest. Thoughts?

Reply:
Good point. I am also unaware of a way to simulate the iframe as being loaded, through Jest, without requiring some refactoring. So, for now, we can bypass this test 👍